doc/poi_mutation_example.graphql

```diff
@@ -207,6 +208,7 @@ mutation {
 			timeTo
 			open
 			sortNumber
+      useYear
```

Comment:
the indentation looks a bit off here, should be in line with the other params, right?

Reply:
Yes, you're right. It seems like there might be a bug. In my file, the indentation appears correct. I even cloned the repository again, checked out that branch, and verified it. Why is it like that?

Reply:
In a different tool I can see the same: 

![image](https://github.com/smart-village-solutions/smart-village-app-mainserver/assets/1942953/41e6f88c-05a9-400e-ad1b-567ae384c4ff)

I will checkout on my side to have a closer look.

Reply:
If I do something and change the file the whole file gets reindented. So I guess that the file was created externally or with an old setup. Let's keep it as it is for now.